components/cellery-component-test/pom.xml

```diff
@@ -0,0 +1,266 @@
+                                    <artifactId>jackson-dataformat-yaml</artifactId>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>com.fasterxml.jackson.core</groupId>
+                                    <artifactId>jackson-databind</artifactId>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>com.fasterxml.jackson.core</groupId>
+                                    <artifactId>jackson-annotations</artifactId>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>io.cellery</groupId>
+                                    <artifactId>cellery</artifactId>
+                                </artifactItem>
+                            </artifactItems>
+                            <outputDirectory>
+                                ${project.build.directory}/extracted-distributions/ballerina-zip/ballerina-${ballerina.lang.version}/bre/lib
+                            </outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>verify</phase>
+                    </execution>
+                </executions>
+                <configuration>
+                    <argLine>${jacoco.agent.argLine}</argLine>
+                    <systemPropertyVariables>
+                        <ballerina.pack>
+                            ${project.build.directory}/extracted-distributions/ballerina-zip/ballerina-${ballerina.lang.version}/bin
+                        </ballerina.pack>
+                        <sample.dir>
+                            ${project.build.directory}/../../../samples
+                        </sample.dir>
+                        <jacoco.agent.argLine>${jacoco.agent.argLine}</jacoco.agent.argLine>
+                    </systemPropertyVariables>
+                    <suiteXmlFiles>
+                        <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
+                    </suiteXmlFiles>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <goals>
+                    <goal>report</goal>
+                </goals>
+                <configuration>
+                    <dataFile>${project.build.directory}/coverage-reports/jacoco.exec</dataFile>
+                </configuration>
+            </plugin>
+            <plugin>
```

Comment:
Please move this and plugin and the same plugin in "lang" module to a pluginManagement section in the root POM